src/main/scala/esmeta/analyzer/TypeAnalyzer.scala

```diff
@@ -318,5 +318,13 @@ object TypeAnalyzer {
     paramType: Boolean = true,
     returnType: Boolean = true,
     uncheckedAbrupt: Boolean = false,
+    invalidAstProperty: Boolean = true,
+    invalidStrProperty: Boolean = true,
+    invalidNameProperty: Boolean = true,
+    invalidCompProperty: Boolean = true,
+    invalidRecordProperty: Boolean = true,
+    invalidListProperty: Boolean = true,
+    invalidSymbolProperty: Boolean = true,
+    invalidSubMapProperty: Boolean = true,
```

Comment:
Indeed, it's a bit verbose. However, I think that we need a more granular configuration in TypeAnalyzer. It would be best to manage all of it through a single or just a few command-line options.

Reply:
Is there a situation where we should distinguish between them? A concrete example (which requires 'granular' configuration might help.